examples/java/pom.xml

```diff
@@ -327,24 +327,21 @@
   </profiles>
 
   <build>
-    <pluginManagement>
-      <plugins>
-        <!-- BEAM-933 -->
-        <plugin>
-          <groupId>org.codehaus.mojo</groupId>
-          <artifactId>findbugs-maven-plugin</artifactId>
-          <configuration>
-            <skip>true</skip>
-          </configuration>
-        </plugin>
-      </plugins>
-    </pluginManagement>
-
     <plugins>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
       </plugin>
 
+      <!-- BEAM-933 -->
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
+        </configuration>
+      </plugin>
+
       <plugin>
```

Comment:
The current pattern, I think, was designed for `pom.xml` being as easy to skim as possible. The files are so long... We currently have (or aim for) a pattern that

1. all configuration appears in `pluginManagement`,
1. you look at the `plugins` section just to see which plugins are uniquely/newly active in this module and its children.

What do you think?

Separately, can you comment on the warnings?